src/components/virtualRepeat/virtualRepeater.js

```diff
@@ -56,6 +56,14 @@ function virtualRepeatContainerTemplate($element, $attrs) {
     '</div></div>';
 }
 
+/**
+ * Maximum size, in pixels, that can be explicitly set to an element. The actual value varies
+ * between browsers, but IE11 has the very lowest size at a mere 1,533,917px. Ideally we could
+ * *compute* this value, but Firefox always reports an element to have a size of zero if it
+ * goes over the max, meaning that we'd have to binary search for the value.
+ * @const {number}
+ */
+var MAX_ELEMENT_SIZE = 1533917;
```

Comment:
It's not global ; all of the files get wrapped by an immediately invoked function
(otherwise VirtualRepeatController etc. would all be global vars)